Src/VimCore/Vim.fs

```diff
@@ -664,6 +670,10 @@ type internal Vim
 
     member x.MaybeLoadFiles() =
 
+        if x.AutoLoadDigraphs && not _digraphsAutoLoaded then
+            DigraphUtil.AddToMap _keyMap DigraphUtil.DefaultDigraphs

```

Comment:
I think we should create a separate map for digraphs vs. storing them into the standard key map. The aren't a mode and don't have some of the odd states that keyremapmodes have for key resolution